enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Visualization.tsx

```diff
@@ -38,23 +33,19 @@ export const QuestionVisualization = () => {
   const card = question.card();
 
   return (
-    <Box w="100%" h="100%" ref={ref}>
-      <Box w={width} h={height}>
```

Comment:
I think this doesn't work when the chart type sidebar is collapsed using the eye button - the height of the viz now comes from the chart sidebar. Once the sidebar's gone, the visualization height is also gone.

![CleanShot 2567-10-10 at 22 31 49@2x](https://github.com/user-attachments/assets/d2d7a652-4da3-4617-9cf0-f645c4d53c00)


![CleanShot 2567-10-10 at 22 31 29@2x](https://github.com/user-attachments/assets/5e88b10c-6809-443e-ae53-506f073f5e16)

Reply:
@heypoom I've pushed a few changes. I've created a `FlexibleSize` component that should work with everything. I've added it to the `InteractiveQuestion` component and I'll add it to the `QuestionEditor` component now

Reply:
Awesome, works now! [Tested in Shoppy with screenshots in this comment.](https://github.com/metabase/metabase/pull/47744#issuecomment-2408049037)